### ISSUE TITLE
Error handling on login code in onboarding

### ISSUE THREAD

**Issue by michiQ** [yadaguru/yadaguru-app#103]
My bad - didn't think of this one.

When we login it's possible we enter a wrong log in code.  I hadn't mocked up the error handling.  So let's do this.

If we enter in the wrong code we get the popup.

![screen shot 2016-11-11 at 8 34 19 am](https://cloud.githubusercontent.com/assets/8203279/20216681/e45a85f8-a7e9-11e6-8188-cb941e691ba2.png)

If I click 'let me try again'. I go back to the screen to enter the code.

If I click 'resend it' then we go to the 'check your device pop up'.

![screen shot 2016-11-11 at 8 38 09 am](https://cloud.githubusercontent.com/assets/8203279/20216730/41ecccda-a7ea-11e6-9dd2-d3270c208598.png)

**Comment by bobderrico80** [comment]
#81 is a general error-handling issue. Right now, any error is silently logged to the console. The wrong code is definite a special case that we want to handle. I will reference this issue in #81.
